packages/create-remix/templates/arc-stack/README.md

```diff
@@ -57,6 +58,7 @@ You can find the CloudFormation template that Architect generated for you in the
 
 To find it on AWS, you can search for [CloudFormation][cloudformation] (make sure you're looking at the correct region!) and find the name of your stack (the name is a PascalCased version of what you have in `app.arc`, so by default it's RemixAwsStackStaging) that matches what's in `app.arc`, you can find all of your app's resources under the "Resources" tab.
 
+[aws_cli]: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
```

Comment:
We should have all of these links inline.

In the docs we don't want to do that because it's easier to do translations if we don't.

But here, most people will be looking at the source in their editor and we don't want them to have to scroll down to the bottom to find the link. We're not going to do translations of this either.